src/main/kotlin/net/atos/zac/app/documentcreation/DocumentCreationRestService.kt

```diff
@@ -9,98 +9,95 @@ import jakarta.inject.Inject
 import jakarta.inject.Singleton
 import jakarta.validation.Valid
 import jakarta.ws.rs.Consumes
+import jakarta.ws.rs.FormParam
 import jakarta.ws.rs.POST
 import jakarta.ws.rs.Path
+import jakarta.ws.rs.PathParam
 import jakarta.ws.rs.Produces
+import jakarta.ws.rs.QueryParam
 import jakarta.ws.rs.core.MediaType
 import net.atos.client.zgw.zrc.ZrcClientService
-import net.atos.client.zgw.ztc.ZtcClientService
-import net.atos.zac.app.documentcreation.converter.RestDocumentCreationConverter
 import net.atos.zac.app.documentcreation.model.RestDocumentCreationAttendedData
 import net.atos.zac.app.documentcreation.model.RestDocumentCreationAttendedResponse
-import net.atos.zac.app.documentcreation.model.RestDocumentCreationUnattendedData
-import net.atos.zac.app.documentcreation.model.RestDocumentCreationUnattendedResponse
-import net.atos.zac.app.exception.InputValidationFailedException
-import net.atos.zac.app.informatieobjecten.EnkelvoudigInformatieObjectUpdateService
-import net.atos.zac.configuratie.ConfiguratieService
 import net.atos.zac.documentcreation.DocumentCreationService
 import net.atos.zac.documentcreation.model.DocumentCreationDataAttended
-import net.atos.zac.documentcreation.model.DocumentCreationDataUnattended
 import net.atos.zac.policy.PolicyService
 import net.atos.zac.policy.PolicyService.assertPolicy
 import nl.lifely.zac.util.AllOpen
 import nl.lifely.zac.util.NoArgConstructor
+import java.util.UUID
 
 @Singleton
-@Path("documentcreation")
-@Consumes(MediaType.APPLICATION_JSON)
```

Comment:
Why did you remove these from the class level? Shouldn't the other two methods also use these?

Reply:
Bizarre..  But then can you not add `@Consumes(MediaType.TEXT_PLAIN)` to those endpoints? Good to make clear which mediatype we consume.